en/docs/administer/configuring-a-read-only-ldap-user-store.md

```diff
@@ -2,23 +2,21 @@
 
 User management functionality is provided by default in all WSO2
 Carbon-based products and is configured in the
-`         user-mgt.xml        ` file found in the
-`         <IS_HOME>/repository/conf/        ` directory. This file
-is shipped with user store manager configurations for all possible user
-store types (JDBC, read-only LDAP/Active Directory, read-write LDAP and
-read-write Active directory). The instructions given below explains how
-to configure a read-only LDAP or Active Directory as the primary user
-store for the WSO2 server.
+`         deployment.toml       ` file found in the
+`         <IS_HOME>/repository/conf/        ` directory. This
+documentation explains the process of setting up a primary user store
+for your system.
 
 !!! info "The default User Store"
-    The primary user store that is configured by default in the user-mgt.xml
-    file is a JDBC user store, which reads/writes into the internal database
-    of the product server. By default, the internal database is H2 for all
-    WSO2 products excluding the Identity Server. This database is used by
-    the Authorization Manager (for user authentication information) as well
-    as the User Store Manager (for defining users and roles). In the case of
-    the WSO2 Identity Server, the default user store is an LDAP (Apache DS)
-    that is shipped with the product.
+    The primary user store that is configured by default in WSO2 Identity Server
+    is an LDAP user store, This database is used by the Authorization Manager (for user authentication
+    information) as well as the User Store Manager (for defining users and
+    roles).
+
+We can specify the type of user store that we are using in the `         deployment.toml       ` file. 
+If we specified it as a JDBC user store by default,  For detailed guide on configuring 
```

Comment:
If we specified it as a JDBC user store by default,_________. Sentence seems incomplete.